src/sentry/static/sentry/app/views/alerts/rules/details/timeline.tsx

```diff
@@ -24,61 +23,17 @@ import {
 } from 'app/views/alerts/types';
 import {IncidentRule} from 'app/views/settings/incidentRules/types';
 
-type Activities = Array<ActivityType>;
-
 type IncidentProps = {
   api: Client;
   orgId: string;
   incident: Incident;
   rule: IncidentRule;
 };
 
-type IncidentState = {
-  loading: boolean;
-  error: boolean;
-  activities: null | Activities;
-};
-
-class TimelineIncident extends React.Component<IncidentProps, IncidentState> {
-  state: IncidentState = {
-    loading: true,
-    error: false,
-    activities: null,
-  };
-
-  componentDidMount() {
-    this.fetchData();
-  }
-
-  componentDidUpdate(prevProps: IncidentProps) {
-    // Only refetch if incidentStatus changes.
-    //
-    // This component can mount before incident details is fully loaded.
-    // In which case, `incidentStatus` is null and we will be fetching via `cDM`
-    // There's no need to fetch this gets updated due to incident details being loaded
-    if (
-      prevProps.incident.status !== null &&
-      prevProps.incident.status !== this.props.incident.status
-    ) {
-      this.fetchData();
-    }
-  }
-
-  async fetchData() {
-    const {api, orgId, incident} = this.props;
-
-    try {
-      const activities = await fetchIncidentActivities(api, orgId, incident.identifier);
-      this.setState({activities, loading: false});
-    } catch (err) {
-      this.setState({loading: false, error: !!err});
-    }
-  }
-
+class TimelineIncident extends React.Component<IncidentProps> {
   renderActivity(activity: ActivityType, idx: number) {
     const {incident, rule} = this.props;
-    const {activities} = this.state;
-    const last = this.state.activities && idx === this.state.activities.length - 1;
+    const last = incident.activities && idx === incident.activities.length - 1;
```

Comment:
could we destructure `activities` out of `incident` so we don't have to type `incident.activities` as much?